nano/node/rpc.cpp

```diff
@@ -3531,6 +3531,7 @@ void nano::rpc_handler::wallet_representative_set ()
 {
 	rpc_control_impl ();
 	auto wallet (wallet_impl ());
+	bool update_existing_accounts (request.get<bool> ("update_existing_accounts", false));
```

Comment:
Very minor: this line could be moved to a more narrow scope, inside `if (!representative.decode_account (representative_text))`

Other than that, LGTM

Reply:
Thx